src/typings/payouts/responseAdditionalDataCommon.ts

```diff
@@ -555,13 +555,6 @@ export namespace ResponseAdditionalDataCommon {
         Green = <any> 'GREEN',
         Fraud = <any> 'FRAUD'
     }
-    export enum MerchantAdviceCodeEnum {
```

Comment:
This might be a breaking change if somebody is using this enum on a released version.